src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
Ok, since we want this to be practical short term, I'll just run w the flag for today. Beyond that we need to think on it.